Converted a float to int

vvkrddy opened the pull request:
While the _num_ parameter in np.linspace() is an integer, here a float is being passed.

Commit:
Converted a float to int

While the 'num' parameter in np.linspace() is an integer, here a float is being passed.

Commit:
Corrections

Changed 3 lines

Commit:
Merge pull request #1 from vvkrddy/master

Corrections

wegenmat-privat commented:
HI @vvkrddy,

thank for your contribution.
After short review and test, your changes will be merged.

Best Matthias

Commit:
Prevents RuntimeWarning

Prevents 'divide by zero' runtime warning by not passing 0 to np.log10() in line 167

Commit:
Reverted back to previous commit

Reverted back to the last commit removing changes done to prevent runtime warning
